coapthon/client/coap.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, server, starting_mid, callback, sock=None, cb_ignore_read_exc
         else:
             self._socket = socket.socket(socket.AF_INET6, socket.SOCK_DGRAM)
             self._socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+            self._socket.setsockopt(socket.IPPROTO_IPV6, socket.IPV6_MULTICAST_HOPS, 5)
```

Comment:
This is a legitimate change to the library that is necessary for OCF over Thread. The problem we've run into is that by default, the multicast hop limit is 1, which does not make it past the Border Router. This change is necessary for the same reason that the multicast hop limit in e.g. the CTT & DeviceSpy needs to be changed.